api/resourcecreator.go

```diff
@@ -607,6 +620,63 @@ func createService(deploymentRequest NaisDeploymentRequest, k8sClient kubernetes
 	return createServiceResource(serviceDef, deploymentRequest.Namespace, k8sClient)
 }
 
+func createRedisDef(deploymentRequest NaisDeploymentRequest) *redisapi.RedisFailover {
+	replicas := int32(3)
+	resources := redisapi.RedisFailoverResources{
+		Limits:   redisapi.CPUAndMem{Memory: "100Mi"},
+		Requests: redisapi.CPUAndMem{CPU: "100m"},
+	}
+	if deploymentRequest.FasitEnvironment != ENVIRONMENT_P {
+		replicas = int32(1)
+		resources = redisapi.RedisFailoverResources{
+			Limits:   redisapi.CPUAndMem{Memory: "50Mi"},
+			Requests: redisapi.CPUAndMem{CPU: "50m"},
+		}
+	}
+
+	spec := redisapi.RedisFailoverSpec{
+		HardAntiAffinity: false,
```

Comment:
Can we label the CRD with a value=app so that when listing all k8s resources belonging to an application the CRD is listed as well.?

Reply:
Sure, didn't know about that.